www/src/pages/faq.md

```diff
@@ -4,11 +4,130 @@ pageTitle: FAQ
 layout: ../layouts/Markdown.astro
 ---
 
-- Is this ready to use?
-- What's the license?
-- How do I give feedback?
-- How do I get involved?
-- What tools support this?
-- How does this relate to other token approaches?
-- What is the relationship between JSON Schema and DTCG?
-- [Additional questions from community]
+## Is this ready to use?
+
+The Design Tokens Community Group (DTCG) specification is still in draft status as of October 2025, but it's closer than ever to reaching v1.0.
```

Comment:
We are going to use a calendar date as the first version. This probably doesn’t make much sense, so maybe add “first stable version of _____” or whatever (not particular on wording)

```suggestion
The Design Tokens Community Group (DTCG) specification is still in draft status as of October 2025, but it's closer than ever to reaching 2025.10.
```